src/const.ts

```diff
@@ -1,17 +1,19 @@
-export const AppRoute = {
-  MAIN: 'main',
-  LOGIN: 'login',
-  FAVORITES: 'favorites',
-  OFFER: 'offer',
-  DEFAULT_CITY: 'Paris'
+export const maxImagesInRoomPage = 6;
+
+export enum AppRoute {
+  MAIN = 'main',
+  LOGIN = 'login',
+  FAVORITES = 'favorites',
+  OFFER = 'offer',
+  DEFAULT_CITY = 'Paris'
 };
 
-export const APIRoute = {
-  HOTELS: '/hotels',
-  FAVORITE: '/favorite',
-  COMMENTS: '/comments',
-  LOGIN: '/login',
-  LOGOUT: '/logout',
+export enum APIRoute {
```

Comment:
```suggestion
export const enum APIRoute {
```
https://www.typescriptlang.org/docs/handbook/enums.html#const-enums

Reply:
enum это выразительный подход. (+) тебе. и почти всегда к нему стоит добавить const.